src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/response/CreatedByResponseDTO.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CreatedByResponseDTO {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("email")
+    private String email;
+
+    @JsonProperty("full_name")
+    private String fullName;
+
+    @JsonProperty("share_identity")
+    private boolean shareIdentity;
+
+    public CreatedByResponseDTO() {
```

Comment:
The SonarLint IntelliJ plugin has highlighted the empty constructor for No empty methods allowed. Classes will have an empty public constructor by default if you don't include one.

Reply:
Could also remove the empty constructor from ObjectionResponseDTO while you're at it

Reply:
Removed constructor for CreatedByResponseDTO, the one for the ObjectionResponseDTO is being used by the POST method.